tests/nemo_text_processing/ar/data_text_normalization/test_cases_money.txt

```diff
@@ -0,0 +1,26 @@
+$7~سبعة دولار
+$7.50~سبعة دولار وخمسون سنت
+$7.05~سبعة دولار وخمسة سنتات
+$7.99~سبعة دولار وتسعة وتسعون سنت
+$7.10~سبعة دولار وعشرة سنتات
+$0.99~تسعة وتسعون سنت
```

Comment:
what about other classes like date, time, whitelist etc?

Reply:
Date/Time are not supported. Whitelist, will add them.